test/unit/extract/csv_test.py

```diff
@@ -55,12 +55,17 @@ def test_load_source(mock_pd):
 
 
 def test_extract():
-    extractor = FakeExtractor()
+    class SuperFakeExtractor(FakeExtractor):
+        def process_raw(self, df, page, **partition):
+            return df
+
+    extractor = SuperFakeExtractor()
     # Create a sample of data we could expect from an EIA CSV
     company_field = "company"
     company_data = "Total of All Companies"
     df = pd.DataFrame([company_data])
     df.columns = [company_field]
+
     # TODO: Once FakeExtractor is independent of eia176, mock out populating _column_map for PARTITION_SELECTION;
     #  Also include negative tests, i.e., for partition selections not in the _column_map
     with (
```

Comment:
Instead of dropping the rename, let's actually test it.

```suggestion
    with (
        patch.object(CsvExtractor, "load_source", return_value=df),
        patch.object(
            # Testing the rename
            GenericMetadata,
            "get_column_map",
            return_value={company_field: 'company_rename'},
        ),
        patch.object(
            # Transposing the df here to get the orientation we expect get_page_cols to return
            CsvExtractor,
            "get_page_cols",
            return_value=df.T.index,
        ),
    ):
        res = extractor.extract(**PARTITION)
    assert len(res) == 1 # Assert only one page extracted
    assert list(res.keys()) == [PAGE] # Assert it is named correctly
    assert res[PAGE]['company_rename'][0] == company_data # Assert that column correctly renamed and data is there.

```